### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by fvasco**

### CODE REVIEW

File: LuaMenu/widgets/chobby/i18n/chililobby.lua [hunk 1]
```diff
@@ -506,6 +506,262 @@ return {
...
+		time_to_respond = "Tempo di risposta",
+		are_you_ready = "Sei pronto?",
+		seconds_short = "s",
+		not_ready_leaving_queue = "Non pronto, abbandono la coda",
+		waiting_for_other_players = "In attesa di altri giocatori",
+		game_starting_soon = "Gioco che sta per iniziare",
+		timeout_leaving_queue = "Tempo scaduto, abbandono la coda",
+			one = "Un ultimo elemento da scaricare.",
+			other = "%{count} elementi rimasti da scaricare.",
+		},
```
Comment (sprunk) [inline LuaMenu/widgets/chobby/i18n/chililobby.lua:670]: did something break here? there is an unmatched `}` bracket.

File: LuaMenu/widgets/chobby/i18n/chililobby.lua [hunk 2]
```diff
@@ -506,6 +506,262 @@ return {
...
+		technology = "Tecnologia",
+		options = "Opzioni",
+		codex = "Codex",
+		technology = "Tecnologia",
+
+		-- gui_rank_update_window.lua
+		rank_gained = "Grado Acquisito",
+		rank_lost = "Grado Perso",
+
+		advanced_button_tooltip = [[Fully configure game settings. Activate automatic advanced mode by disabling "Simple skirmish setup" in lobby settings.]],
```
Comment (sprunk) [inline LuaMenu/widgets/chobby/i18n/chililobby.lua:749]: looks untranslated

File: LuaMenu/widgets/chobby/i18n/chililobby.lua [hunk 3]
```diff
@@ -506,6 +506,262 @@ return {
...
+		--
+		start_download = 'Inizia download',
+		download_noun = 'Download',
+		friends = 'Amici',
+		start_verb = 'Avvia',
+		open_mp_game = "Ospita un gioco",
+		steam_friends = "Amici di Steam",
+		host = "Host",
+		match_found = "Corrispondenza trovata",
+		match_found_quickplay = "Small Teams available",
```
Comment (sprunk) [inline LuaMenu/widgets/chobby/i18n/chililobby.lua:572]: looks untranslated

**Review by sprunk — CHANGES_REQUESTED** [review]
Sorry for the late review! Looks decent overall, but there are some things to fix (left specific comments under the relevant lines). Also there needs to be an entry in the list of languages for it to show up on the list, here: https://github.com/ZeroK-RTS/Chobby/blob/d2ab5342a14535e47f854e72218033bf3ada2d7b/LuaMenu/widgets/chobby/components/configuration.lua#L254-L257

**Comment by fvasco** [comment]
Thank you for review.
I updated files.

**Comment by sprunk** [comment]
Thanks!

**Pull request merged**